examples/website/trips/app.js

```diff
@@ -76,7 +76,7 @@ export class App extends Component {
         getPath: d => d.segments,
         getColor: d => (d.vendor === 0 ? [253, 128, 93] : [23, 184, 190]),
         opacity: 0.3,
-        strokeWidth: 2,
+        strokeWidth: 5,
```

Comment:
Why do you change this?

Reply:
to make the path wider so that the rendering effect looks more substantial